src/components/PhoneTextInput/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Why can't this be at `src/components/PhoneTextInput.js`?

Reply:
Sure, we can do that.